content/en/about/case-studies/rappi/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Rappi Manages Growing Pains with Istio"
+linkTitle: "Rappi Manages Growing Pains with Istio"
+quote: "Istio seamlessly gave our team the ability to monitor services at any scale." 
+author:
+    name: "Ezequiel Arielli"
+    image: "/img/authors/ezequiel-arielli.jpg"
+companyName: "Rappi"
+companyURL: "https://www.rappi.com/"
+logo: "/logos/rappi.png"
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 30
```

Comment:
@craigbox Again, a question on weighting. If we keep increasing the number, then the new ones will be at the bottom, potentially requiring a bunch of scrolling.

Reply:
@pymander 52 please

Reply:
@craigbox Done!